pkg/optimizer/problem_data.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package optimizer
```

Comment:
We cannot place this code in `manager/controllers/app` because it will create a circular dependency: optimizer will have to import `app` package to use this code, and `app` package will have to import `optimizer` package to use the optimizer.
We can place this code in a package of its own. @shlomitk1 , what do you think?

Reply:
Perhaps a package called `datapath` ?

Reply:
Moved the code to a new package called `datapath`